Run.scala

```diff
@@ -0,0 +1,13 @@
+object Run {
+   implicit class IntTimes(x: Int) {
+      def times [A](f: =>A): Unit = {
+         def loop(current: Int): Unit =
+         
+         if(current > 0){
+            f
+            loop(current - 1)
```

Comment:
when possible use for loops or at least add the `@tailrec` annotation  
https://www.scala-lang.org/api/2.12.3/scala/annotation/tailrec.html